edx_solutions_api_integration/sessions/test_login_ratelimit.py

```diff
@@ -68,8 +68,8 @@ def test_login_ratelimiting_unblock(self):
         message = _('Rate limit exceeded in api login.')
         self._assert_response(response, status=403, message=message)
 
-        # now reset the time to 5 mins from now in future in order to unblock
-        reset_time = datetime.now(UTC) + timedelta(seconds=300)
+        # now reset the time to 9 mins from now in future in order to unblock
+        reset_time = datetime.now(UTC) + timedelta(seconds=560)
```

Comment:
@wasifarbisoft has the reset time increased from 5 minutes to 9 minutes?

Reply:
@msaqib52 
Yes by increasing the reset time, apis that were calling do_post got ample time to respond before reset threshold reach